### ISSUE TITLE
Accommodation in Taipei

### ISSUE THREAD

**Issue by serapath** [g0v/taipei#3]
Hey, very useful information, thx for sharing.

@ninabreznik and me are from Berlin and connected to the Open Knowledge Foundation and just arrived 2 days ago for attending g0v meetups and hackathons.

We are very inspired by g0v and some talks given by audrey tang and also follow the dat-data.com team for quite some time, which is where i first recognized @clkao. We would love to learn more about g0v zero and contribute. We plan to stay at least 6 weeks, maybe more and are currently looking for very affordable accommodation.

If someone could point us into the right direction about where to stay - ideally around people that share the mindset, maybe some couchsurfing profiles or similar, that would be very appreciated :-)

**Comment by serapath** [comment]
We live on a very minimal budget in general and hope to be able to find a place to stay for less than 400 EUR per months - which is what we pay in germany for 50 m². After asking and searching for a bit, we had the most success asking here:
- https://www.facebook.com/groups/754849567895453/
- https://www.facebook.com/groups/RoomShareTaiwan/
- https://www.facebook.com/groups/Taiwanapartmentsandroommates/
- https://www.facebook.com/groups/taipei.taiwan.apartment.rentals
- http://www.homeaway.tw/
- https://www.roomster.com/
- https://www.airbnb.com/

Using a [translation plugin](https://chrome.google.com/webstore/detail/google-translate/aapbdbdomjkkjkaonfhkkikfgjllcleb/related?hl=en) for the chrome browser 
- https://rent.housefun.com.tw/
- calling http://www.ezstay.com.tw/
- https://rent.591.com.tw/new/

And we explore some other options, like having like minded people that help us and we try to return the favor by helping out where we can
- https://www.workaway.info/
- https://www.couchsurfing.com/

So if this is a meaningful addition to the readme, i can create a pull request too

**Comment by ninabreznik** [comment]
I would also add www.digiquarters.com where we are currently staying :)

**Comment by clkao** [comment]
Thanks!  As the document was originally intended for short visits, but I can see this being super useful. It might be worthwhile to put into a separate "Extended Stay" file though, as this should now also include general instructions like dumping trash and going to clinics/hospitals.

In general, I think we should avoid putting specific rental offering, and limit to listing services. otherwise this will soon be filled with rental ads.

**Comment by ninabreznik** [comment]
True. Didn't even think of that, but makes total sense. Hope to see you on Thursday on g0v zero meetup 👍
